addons/mass_mailing/static/tests/mass_mailing_favourite_filter_tests.js

```diff
@@ -152,6 +152,9 @@ QUnit.module('favorite filter widget', {
         assert.isNotVisible(form.$('.o_mass_mailing_save_filter_container'),
             "should not have option to save filter if filter is already set");
         await testUtils.dom.click('.o_field_many2one[name="mailing_filter_id"] input');
+        // When the popup is shown, the input (.o_mass_mailing_filter_name) field is focussed. Adding preventScroll=true
```

Comment:
Sorry for being a grammar nazi but "focused" takes only one "s". Nice to have added the comment though :+1:
I don't understand what this tests though, you're checking if the input is still in view?

Reply:
Thanks, I have changed it. Actually I didn't write this test. I have just added an additional click because the test was failing after the correction I have done (filterInput.focus(); --> filterInput.focus({ preventScroll: true });). I know it is a hack but I couldn't find something else until now ...
The test checks that after having added a new filter, when we click on the drop down to display the filters, there is the right number of filters: 1 that was at the beginning + one we just added. It does it by counting the "li" of the list of filter.

Reply:
Thanks for your answer. That seems weird, do you know why your change breaks the test and why triggering an extra click makes it pass? Are you sure the fail was a false negative? Are you sure it's not a question of synchronicity that could potentially lead to non-deterministic fails (in other words, does a `await nextTick()` after the original click fix it too)?

Reply:
Thanks for your comments. Yes of course a nondeterministic test would be very bad. I have tested with await nextTick() instead of the second click and the test fails so a second click is needed in the test (while not needed when testing in the browser). I must admit I have no clue why unfortunately.